spec/models/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.new(title: 'My first post', text: 'This is my first post',
+             author_id: 1, comments_counter: 1, likes_counter: 1)
+    before { subject.save }
+
+    it 'title should be present' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'text should be present' do
+      subject.text = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'author_id should be present' do
+      subject.author_id = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'comments_counter should be present' do
+      subject.comments_counter = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'likes_counter should be present' do
+      subject.likes_counter = nil
```

Comment:
- [ ] [OPTIONAL] For comments and likes counters, you can include testing if the value is less than zero, or the value is zero... this will test the validations in the models

Reply:
Thank you